jupyter-notebooks/api_guides/features_api/planet_sdk_features_demo.ipynb

```diff
@@ -0,0 +1,742 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d8f60483",
+   "metadata": {},
+   "source": [
+    "___"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "40384055",
+   "metadata": {},
+   "source": [
+    "### Deleting Collections\n",
+    "\n",
+    "Using the `collection_id` you may delete a collection using `delete_collection()`. Be careful when you do this, as there is no confirmation output from this method, and the collection will just be deleted."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b61da442",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "coll_id = ''\n",
+    "pl.features.delete_collection(coll_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5d2b33e6",
+   "metadata": {},
+   "source": [
+    "_____"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "861cc6e6",
+   "metadata": {},
+   "source": [
+    "# Using Feature References as input geometries\n",
+    "\n",
+    "The `'pl:ref'` property of a feature is used in other Planet APIs to link to the geometry of the feature instead of having the user track bounding box and geometry variables. Planet `Subscriptions`, `Orders`, and `Data` APIs are all able to make use of these pl feature reference links. \n",
+    "\n",
+    "PL ref links are submitted in the `geometry` argument in API calls as a dictionary: \n",
+    "```python\n",
+    "geometry = {\n",
+    "    'type': 'ref',\n",
+    "    'content': 'pl:features/my/<collection_id>/<item_id>'\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "Provided below is consolidated code to use the SDK to get a `pl:ref` id from the features API, and then input that id into an order request. If you are unfamiliar with the mechanics of the `order_request` functions of the SDK, there is a full notebook, `planet_sdk_orders_demo.ipynb` located in `api_guids/orders_api/` that will walk you through how to use the Planet Python SDK to create and submit Orders."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "id": "c70b7db8",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'type': 'ref', 'content': 'pl:features/my/cairo_areas-orELyjq/0-m2z6LNQ'}"
+      ]
+     },
+     "execution_count": 29,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "### SDK Order workflow\n",
+    "## example: get the first Collection you have, and use the id of its first item as a geometry in an Order\n",
+    "\n",
+    "# list all your collections\n",
+    "collections = pl.features.list_collections()\n",
+    "collections_list = [collection for collection in collections]\n",
+    "\n",
+    "# Get the id of the first collection in your list\n",
+    "collection_id = collections_list[0]['id']\n",
+    "\n",
+    "collection_items = list(pl.features.list_items(collection_id))\n",
+    "feature_ref = collection_items[1]['properties']['pl:ref']\n",
```

Comment:
Any reason you're using the second feature in the collection? This should maybe be mentioned in the description above.
And since it is the second feature, this is going to frequently fail if they use or have created a different collection